packetbeat/pb/event.go

```diff
@@ -35,6 +35,13 @@ import (
 // event at publish time.
 const FieldsKey = "_packetbeat"
 
+// Network direction values.
+const (
```

Comment:
We should have these in the ECS go code ;-)

Reply:
I agree. 👍